meld/version.py

```diff
@@ -1,4 +1,3 @@
 # Copyright (C) 2020 Krishnaswamy Lab, Yale University
 
-__version__ = "0.3.2"
-
+__version__ = "0.4.0"
```

Comment:
If you're going to change the function APIs, you should either (and preferably both) bump a major version number to indicate breaking changes, or use legacy_api_wrap to accept the legacy arguments (see https://github.com/theislab/scanpy/blob/master/scanpy/external/pp/_magic.py#L19 for an example).

Reply:
Also any functions with changed names ideally you would created a deprecated function mapping from old to new.

Reply:
Yeah I was wondering about this. I'd rather have the 0.x block reserved for the prepublication beta software and then just break everything for version 1.0 🤣